### ISSUE TITLE
Strange chars in dist/ExifTool.d.ts

### ISSUE THREAD

**Issue by apolkingg8** [mceachen/exiftool-vendored.js#16]
Hi, I found some strange chars in `dist/ExifTool.d.ts`:
```js
/**
 * Use this singleton rather than instantiating new ExifTool instances in order
 * to leverage a single running ExifTool process. As of v3.0, its `maxProcs` is
 * set to the number of CPUs on the current system; no more than `maxProcs`
 * instances of `exiftool` will be spawned. You may want to experiment with
 * smaller or larger values for `maxProcs`, depending on CPU and disk speed of
 * your system and performance tradeoffs.
 *
 * Note that each child process consumes between 10 and 50 MB of RAM. If you
 * have limited system resources you may want to use a smaller `maxProcs` value.
 */
export declare const exiftool: ExifTool;
ol;  // <=This
```
I tried `yarn cache clean` and reinstall it, but it still exist.
The `ol;` seems like... I don't know, maybe your cat on keyboard 😆 ?

**Comment by mceachen** [comment]
Naughty :cat:!

I'll republish. Thanks for the catch.

**Comment by mceachen** [comment]
Published v4.17.0 🎉
